pyproject.toml

```diff
@@ -0,0 +1,28 @@
+[tool.poetry]
+name = "vtpserver"
+version = "0.1.0"
+description = "A Python API server to demonstrate the use of VoteTrackerPlus for a mock election"
+authors = ["Neil Johnson <neil@cadent.com>"]
+license = "GNU General Public License v2.0"
+readme = "README.md"
+packages = [{include = "vtpserver", from = "src"}]
+
+[tool.poetry.dependencies]
+python = "^3.11"
+requests = "^2.28.2"
```

Comment:
Is this used in the sources or only in tests (in which case it should go in the dev dependencies)?

Reply:
As noted elsewhere, I've dropped `requests` from the requirements.